suite-native/module-send/src/components/CryptoAmountInput.tsx

```diff
@@ -26,13 +27,19 @@ export const sendAmountInputWrapperStyle = prepareNativeStyle<{ isDisabled: bool
 export const SendAmountCurrencyLabelWrapper = ({
     children,
     isDisabled,
+    onPress,
 }: {
     children: ReactNode;
     isDisabled: boolean;
+    onPress?: TextInputProps['onPress'];
 }) => {
     const textColor: Color = isDisabled ? 'textSubdued' : 'textDefault';
 
-    return <Text color={textColor}>{children}</Text>;
+    return (
+        <Pressable onPress={onPress}>
+            <Text color={textColor}>{children}</Text>
+        </Pressable>
```

Comment:
Not sure this is worth doing but without it tapping on the label doesn't lead to a switch.